actions/views/root.js

```diff
@@ -13,17 +13,22 @@ const pluginTranslationSources = {};
 
 export function loadConfigAndMe() {
     return async (dispatch) => {
-        // TODO: we will check for graphql here in future
-        // eslint-disable-next-line no-unused-vars
         const [{data: clientConfig}] = await Promise.all([
             dispatch(getClientConfig()),
             dispatch(getLicenseConfig()),
         ]);
 
+        const isGraphQLEnabled = clientConfig && clientConfig.FeatureFlagGraphQL === 'true';
+
         let isMeLoaded = false;
         if (document.cookie.includes('MMUSERID=')) {
-            const dataFromLoadMe = await dispatch(loadMe());
-            isMeLoaded = dataFromLoadMe?.data ?? false;
+            if (isGraphQLEnabled) {
+                const dataFromLoadMeGQL = await dispatch(loadMeGQL());
+                isMeLoaded = dataFromLoadMeGQL?.data ?? false;
+            } else {
+                const dataFromLoadMe = await dispatch(loadMe());
+                isMeLoaded = dataFromLoadMe?.data ?? false;
+            }
```

Comment:
What about something like (here and 2x in packages/mattermost-redux/src/actions/users.ts):

```js
const action = isGraphQLEnabled ? loadMeGQL : loadMe;
const loadData = await dispatch(action());
isMeLoaded = loadData?.data ?? false;
```

And maybe since it's done 3x so far it's worth actually abstracting to a function?

Reply:
We could make it so that that's `loadMe` and then rename the old version to `loadMeRest` or something similar. I guess that could also save us from having to update the existing code that uses `loadMe` as well

Reply:
i changed the rest version to `loadMeREST` and the graphql one to `loadMe`

Reply:
I meant that we could have a dedicated `loadMe` action that includes the choice between these like
```javascript
export function loadMe() {
    return (dispatch, getState) => {
        const isGraphQLEnabled = ...; // Get this from the store or pass it in as an argument to loadMe
        let data;
        if (isGraphQLEnabled) {
            data = await dispatch(loadMeGQL());
        } else {
            data = await dispatch(loadMeREST());
        }
        return data;
    };
}
```
Then we can just call that everywhere this is needed instead of needing to repeat that if-else